bcgov/biohubbc-platform
# 425 + 427 - Show / Download Artifacts + Improved Security Rules

## Links to Jira Tickets

- [425 - Show Documents In Biohub](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-425)
- [427 - Add Placeholder Security Rules](https://apps.nrs.gov.bc.ca/int/jira/browse/SIMSBIOHUB-427)

## Description of Changes

### 425 - Show Documents In Biohub
- Renders documents (artifacts) in Submission views under "Artifact" table
- Per each artifact row, renders a "Download" button
- Flow: user views submission -> clicks download -> endpoint generates signed URL -> opens signed URL in new tab
- Note: admins can bypass the security check for the submission_feature (artifact)

### 427 - Add Placeholder Security Rules
- Seed updated to have realistic security rules
- Submission seed now uses these newly seeded rules

## Testing Notes

- View the submissions from the admin accessible page (Pending Review / Completed)
- Verify the download button rendering for the seeded submissions.
